extensions/kubernetes-client/test/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-kubernetes-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-kubernetes-client-test</artifactId>
+    <name>Quarkus - Kubernetes Client - Test</name>
```

Comment:
I was wondering the same thing to be honest.... But I didn't do it since this one is extremelly specific (as opposed to a need to test against a database).
WDYT?

Reply:
Should we go ahead and live it as it is?

Reply:
OK, I thought about it more and I'd prefer to see it moved to the test framework.

I really prefer not having test related code in the main `extensions` part.

Can you take care of it?

Reply:
Sure thing, I'll fix it in a few hours since I'm not a at a computer right now